examples/resnet50/export.py

```diff
@@ -0,0 +1,61 @@
+import argparse
```

Comment:
Could you add an example command line?

Reply:
@shinh add quick docs on comments.

I added YOLOv2Tiny example. Could you check it? thx.

Reply:
Thanks! If there is no particular reason, I like usage at the beginning of code. Is it common to put the example usage after import lines (perhaps it's specified in pep8)? If it's the case, the current location is fine to me.

Reply:
PEP8 allow top comments before import if the comments follow pydoc style, so I fixed, Thnaks!